File: benchmark/results/python-multilang/run-005/b-baseline/output/metaengine_demo/notification/value_objects/delivery_attempt.py
"""DeliveryAttempt value object for the notification domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class DeliveryAttempt:
    """DeliveryAttempt value object."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str

File: benchmark/results/java-invocation-inline/run-002/a-mcp/output/notification/value_objects/DeliveryAttempt.java
package com.metaengine.demo.notification.value_objects;

import java.time.OffsetDateTime;

// DeliveryAttempt value object.

public class DeliveryAttempt {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public DeliveryAttempt(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}
